src/openeo_gfmap/stac/constants.py

```diff
@@ -0,0 +1,232 @@
+"""
+Constants in the STAC collection generated after a series of batch jobs
+"""
+import pystac
+
+TITLE = "WorldCereal Level 1 cache"
```

Comment:
If this title is something we want a default value for, not sure if this has to be WorldCereal? At this point, functionality should be broader than just WorldCereal extraction use case right?

Reply:
indeed, we should WorldCereal or other concrete references in gfmap

Reply:
I removed them and made them as parameters for the GFMAPJobManager, I agree they shouldn't be hardcoded values